Assets/MixedRealityToolkit.SDK/Experimental/Dialog/Scripts/DialogButtonType.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dialog
+{
+    /// <summary>
+    /// Enum describing the style (caption) of button on a Dialog.
+    /// </summary>
+    public enum DialogButtonType
```

Comment:
Add flags attribute to make it clear
https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netframework-4.8